src/components/Sidebar/components/NavItemExplore.css

```diff
@@ -0,0 +1,11 @@
+.sidebar .categories-section {
+  display: none;
+  list-style-type: none;
+  padding: 0.5rem 0 0 3rem;
+}
+
```

Comment:
Suggestion: you could change this to px as this values are too small to factor in responsiveness using rems